tests/unit/common/test_configuration.py

```diff
@@ -0,0 +1,11 @@
+import os
+from unittest import TestCase, mock
+
+from common import config
+
+
+class TestConfiguration(TestCase):
+
+    @mock.patch.dict(os.environ, {"I_KNOW_KARATE": "show me"})
```

Comment:
We are mixing ' with " in this file. I think we should stick to only one.

Reply:
Changed to `'`